### DISCUSSION TITLE
Does this plane works with not licensed version of the game?

### DISCUSSION THREAD

**Discussion by MrVakoze** [flybywiresim/aircraft#7922] [Help]
Will this mod work with torrent (Not Licensed) version of the game? I am from Russia and i can't buy the game properly so i should use this variant to play in this beautiful game. I have tried this mod already, but i got 0.0v battery bug on 0.9.1 version of the plane. I tried the fix pinned in discussion but it didn't fix it. "Version of the game is (1.12.13.0 Update 10) Repack from HOODLUM." I can't use the version higher than that because i need the "unreal weather metar" to have real weather in the sim. This is the latest version of the game which works with metar weather properly.

**Comment by MrVakoze** [comment]
Okay, i checked the discord and got the answer, mod will work properly only with the latest version of the game. But this brings the another question, what is the latest supported version of mod to this version of the game?

**Comment by tracernz** [comment]
We do not accept or condone piracy in any form. Only the latest stable release of MSFS is supported at any time.
